### PR TITLE
add postgresql scheme

### LINKED ISSUE #966: no loader for url scheme: postgresql
**Small description**

When attempting to start visidata with
```
vd postgresql:///localdb
```
it fails with
```
Error: no loader for url scheme: postgresql
```

**Expected result**

I would have expected it to work the same way 
```
vd postgres:///localdb
```
works, as [both URL schemes are valid](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING).

P.S.: This is somewhere in between a bug and a feature request. As it's super small and about something existing, I called it a bug. I will raise a PR to fix this shortly.

### PR DESCRIPTION
**Pull request by zormit**
This PR adds the `postgresql://` URL scheme as an alias to `postgres://`

### CODE REVIEW

**Comment by zormit** [comment]
note: This is a PR "flying blindly" -- I didn't test this. Still have to work on my dev setup ;)

File: visidata/loaders/postgres.py [hunk 1]
```diff
@@ -52,6 +52,10 @@ def openurl_postgres(url, filetype=None):
     return PgTablesSheet(dbname+"_tables", sql=SQL(conn))
 
 
+def openurl_postgresql(url, filetype):
```
Comment (saulpw) [inline visidata/loaders/postgres.py]: Thanks for putting this together, @zormit !  Let's do `openurl_postgresql=openurl_postgres` and save a line of code :)

Comment (zormit) [reply visidata/loaders/postgres.py]: haha of course

File: visidata/loaders/postgres.py [hunk 2]
```diff
@@ -52,6 +52,10 @@ def openurl_postgres(url, filetype=None):
...
+    return openurl_postgres(url, filetype)
```
Comment (zormit) [inline visidata/loaders/postgres.py]:
```suggestion
openurl_postgresql=openurl_postgres
```

**Comment by zormit** [comment]
@saulpw do you care about a super clean git history? Then I could also squash the two commits again. also i forgot to add a `[tag]` in front of the 2nd commit, whoopsie.

**Review by saulpw — APPROVED** [review]
Looks good, @zormit !   Thanks for putting this together.  I'll squash/merge it now.

**Pull request merged**